### ISSUE TITLE
coverage test upload appears to require a Codecov token

### ISSUE THREAD

**Issue by cjyetman** [RMI-PACTA/pacta.scenario.data.preparation#52]
Not sure how/when/why it stopped working, but it appears that the uploading of test coverage results to Codecov is not happening (even though the workflow passes without error), which then means the badge on the README is out of date.

message in the workflow log says
https://github.com/RMI-PACTA/pacta.scenario.data.preparation/actions/runs/8892294430/job/24416055743#step:5:53
```
[1] "Rate limit reached. Please upload with the Codecov repository upload token to resolve issue. Expected available in 3597 seconds."
```

Documentation for {covr} suggests one can/should use a token through one of a few methods, either explicitly through the `token` argument, or
https://covr.r-lib.org/reference/codecov.html
> a codecov upload token, if NULL then following external sources will be checked in this order:
> 
> the environment variable ‘CODECOV_TOKEN’. If it is empty, then
> 
> package will look at directory of the package for a file codecov.yml. File must have codecov > section where field token is set to a token that will be used.

The existing [codecov.yml](https://github.com/RMI-PACTA/pacta.scenario.data.preparation/blob/main/codecov.yml) here does not have a token specified.

@jdhoffa do you have any idea what's going on here?

**Comment by jdhoffa** [comment]
I'm not sure, I don't recall making a lot of use of the `codecov.yml` action in the past (although I see the utility).

I think the `CODECOV` token would be a good candidate for something to set at the `RMI-PACTA` org level, I can look into creating one

**Comment by cjyetman** [comment]
resolved by #53
